sdks/python/apache_beam/runners/worker/opcounters_test.py

```diff
@@ -19,10 +19,15 @@
 import math
 import random
 import unittest
+from time import sleep
```

Comment:
<!--new_thread; commit:c1c26b3fd7c3a3eba913bfa6b714db6a687754a6; resolved:0-->
Can you just import `time` and use `time.sleep()`?  This is the style in the rest of the codebase.